languages/scala/exercises/concept/operators/.docs/introduction.md

```diff
@@ -0,0 +1,44 @@
+## Operators
+
+Like any other programming language Scala provides all typical operators for native types.
+
+Examples of arithmetic operators:
+
+```scala
+5.3 + 7.1 // Double = 12.4
+5 - 2 // Int = 3
+4L * 8L // Long = 32L
+14 / 4 // Int = 3
+14 % 4 // Int = 2
+14.0 / 4.0 // Double = 3.5
+```
+
+Examples of relational and logical operators:
+
+```scala
+4 > 6 // Boolean = false
+4 == 6 // Boolean = false
+4 != 6 // Boolean = true
+'A' <= 'C' // Boolean = true
+4 != 6 || 4 == 6 // Boolean = true
+4 != 6 && 4 == 6 // Boolean = false
+!(4 > 6) // Boolean = true
+```
```

Comment:
These relational/logical operators are not needed by the student to solve this exercise. I feel like there are two options here:

1. Have a separate exercise on boolean logic and a separate exercise on comparisons and remove them from this exercise.
1. Have this exercise do more and also require the student to work with relational and logical operators.

One thing to consider is that if boolean/relational operators are introduced in this exercise, every exercise that wants to use them would have to take a dependency on this exercise. As this exercise itself depends on `case-classes` and that will depend on `classes` or something like that, I feel that this might become problematic. As such, my suggestion would be to rename this exercise to `operator-overloading` or `custom-operators` or whatever the correct Scala term is, and have separate exercises for the boolean and arithmetic operators.

The above suggestion is also what the C# track does. It has a [`numbers` exercise](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/numbers/) that teaches students about the arithmetic operators (`+`, `-`, etc.) as well as some basic arithmetic comparison operators (`>`, `<=`, `==`, etc.). The [booleans exercise] (https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/booleans/) introduces the `&&` and `||` and `!` boolean operators. Finally, operator overloading is done in a separate exercise.

What do you think, does this make sense?

Reply:
Your suggestion to split it into different exercises makes sense. This "operators are methods" topic from here (there does not seem to be a more specific term for that in Scala, since it is not more than this) might not be needed for someone, who just learns about using existing operators. 
I'd take the C# exercises as template then. :)

Reply:
Great! 👍